docs/docs/job_registries.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "RQ: Job Registries"
+layout: docs
+---
+
+Each queue maintains a set of Job Registries:
+* `StartedJobRegistry` Holds currently executing jobs. Jobs are added right before they are 
```

Comment:
Nitpick: can we make the language here more consistent? Each point should start with `holds ...`

Reply:
Sure, I've added a commit to do this.